pages/price-feeds/use-real-time-data/evm.mdx

```diff
@@ -27,41 +24,22 @@ npm install @pythnetwork/pyth-sdk-solidity
 If you are using Foundry, you will need to create an NPM project if you don't already have one.
 From the root directory of your project, run:
 
-```bash
+```bash copy
 npm init -y
 npm install @pythnetwork/pyth-sdk-solidity
 ```
 
 Then add the following line to your `remappings.txt` file:
 
-```text
+```text copy
 @pythnetwork/pyth-sdk-solidity/=node_modules/@pythnetwork/pyth-sdk-solidity
 ```
 
-### Typescript SDK
-
-Add [pyth-evm-js](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/sdk/js) to your off-chain dependencies.
-
-```bash copy
-npm install --save @pythnetwork/pyth-evm-js
-```
-
 ## Write Contract Code
 
-Add code to your contract to update and read the Pyth price.
-Your contract should do the following four steps:
-
-1. Instantiate the `IPyth` interface from the Solidity SDK using the price feeds [contract address](../contract-addresses/evm).
-1. Select the [Price Feed Ids](https://pyth.network/developers/price-feed-ids) for the assets you want to fetch prices for. Price feeds come in two varieties, Stable and Beta. You should select Stable feed ids
-1. Call `IPyth.getUpdateFee` to calculate the fee charged by Pyth to update the price.
-1. Call `IPyth.updatePriceFeeds` to update the price, paying the fee calculated in the previous step.
-1. Call `IPyth.getPrice` to read the current price, providing the [price feed ID](https://pyth.network/developers/price-feed-ids) that you wish to read.
-
-Simply add this argument to the functions on your contract that need it.
```

Comment:
Maybe move these bullets to the end of the section as a summary of what the snippet does? I think it's not a bad idea to keep an explicit recipe for people to follow (which this is), even if we expect most people to copy/paste the code.